templates/aem-tools/promote-author-standby-to-primary.sh.epp

```diff
@@ -31,4 +31,12 @@ cd <%= $base_dir %>/aem-aws-stack-provisioner/
   --hiera_config conf/hiera.yaml \
   --execute "include aem_curator::action_promote_author_standby_to_primary"
 
+cd <%= $base_dir %>/aem-aws-stack-provisioner/
```

Comment:
The Puppet apply call to `aem_curator::action_promote_author_standby_to_primary` should also have a `translate_exit_code` call. Otherwise exit code 4 and 6 from Puppet would be ignored.